pkg/ebpf/c/tracee.bpf.c

```diff
@@ -656,6 +656,306 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
     return 0;
 }
 
+void __always_inline update_hidden_modules(u64 mod_addr, char *mod_name)
+{
+    if (bpf_map_lookup_elem(&already_reported_hidden_modules_map, &mod_addr) != NULL) {
+        return; // already reported this module - no need to report again
+    }
+
+    kernel_module_t hidden_mod = {.addr = mod_addr};
+    bpf_probe_read_str(hidden_mod.name, MODULE_NAME_LEN, mod_name);
+
+    bpf_map_push_elem(&found_hidden_modules_queue, &hidden_mod, BPF_EXIST);
```

Comment:
Why do we need this queue? Can't you just send an event per module?

Reply:
Yep, I think I can change it to be an event per hidden module